apps/ex_nvr/lib/ex_nvr/model/lpr_event.ex

```diff
@@ -0,0 +1,125 @@
+defmodule ExNVR.Model.LPREvent do
+  use Ecto.Schema
+
+  import Ecto.Query
+
+  alias Ecto.Changeset
+
+  @fields [
+    :capture_time,
+    :plate_number,
+    :direction,
+    :list_type,
+    :confidence,
+    :vehicle_type,
+    :vehicle_color,
+    :plate_color,
+    :device_id
+  ]
+
+  @derive {Flop.Schema,
+           filterable: [:capture_time, :plate_number, :direction, :list_type, :vehicle_type, :vehicle_color, :plate_color, :inserted_at, :device_id],
+           sortable: [:inserted_at, :capture_time, :confidence],
+           default_order: %{
+             order_by: [:inserted_at, :capture_time, :confidence],
+             order_directions: [:desc]
+           },
+           adapter_opts: [
+             join_fields: [
+               device_name: [
+                 binding: :joined_device,
+                 field: :name,
+                 ecto_type: :string
+               ]
+             ]
+           ],
+           pagination_types: [:page],
+           default_limit: 100,
+           max_limit: 150}
+
+  @type t :: %__MODULE__{
+          capture_time: DateTime.t(),
+          plate_number: binary(),
+          direction: binary(),
+          list_type: binary(),
```

Comment:
You should use a enum for the `list_type` and `direction`. 
For direction let's go with `forward`, `reverse`.
For `list_type`: `white`, `black` and `visitor`.